haskell/test/Lorentz/Contracts/Nettest/Nettest.hs

```diff
@@ -1,6 +1,9 @@
 -- SPDX-FileCopyrightText: 2021 Oxhead Alpha
 -- SPDX-License-Identifier: MIT
 
+-- for Ord ImplicitAddressWithAlias
```

Comment:
Perhaps it's worth doing this change in Morley?

Reply:
I also don't feel strongly either way, but I've opened [morley/#936](https://gitlab.com/morley-framework/morley/-/issues/936) to do this, as it doesn't hurt.

If we change our mind that issue can always be closed too